ProjectName/Resources/LaunchScreen.storyboard

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Should we move `LaunchScreen.storyboard` to folder `Resources/LauchScreen/Base.lproj`?

Reply:
I've considered it already but seemed we had only one `LaunchScreen`. So I decided to move `LaunchScreen.storyboard` to `Resources` as recently doing.
But let's see if anyone has the same concern as well.
cc: @suho what do you think?